app/services/description_import.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class DescriptionImport
+  include Dry::Monads[:result]
+
+  def self.import(description:, csv:)
+    new(description: description, csv: csv).import
+  end
+
+  def initialize(description:, csv:)
+    @description = description
+    @csv = csv
+  end
+
+  def import
+    params = @description.to_h
+
+    row = @csv.first
+    # The source_id is only there for the user to reference and should be ignored for data processing
+    headers = row.headers.excluding('source_id')
+
+    headers.each do |address|
+      split_address = address.scan(/[[:alpha:]]+|[[:digit:]]+/)
+                             .map { |item| /\d+/.match?(item) ? item.to_i - 1 : item.to_sym }
+      visit(params, split_address, row[address])
```

Comment:
I'm not sure what this bit of code is for, what is split_address ?

Reply:
perhaps adding this a code comment will help a future developer (i.e. show example input from csv and resulting parsed output)